lib/msal-core/docs/FAQ.md

```diff
@@ -379,6 +381,17 @@ You can read more about this behavior [here](https://docs.microsoft.com/azure/ac
 
 MSAL.js will only process tokens which it originally requested. If your flow requires that you send a user a link they can use to sign up, you will need to ensure that the link points to your app, not the B2C service directly. An example flow can be seen in the [working with B2C](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/working-with-b2c.md) doc.
 
+## I used to be able to get an access token when using acquireTokenSilent with clientId as scope, now I can't. Why?
+
+If you are sending `clientId` as the scope, you are requesting scopes for `openid` and `profile`, which are **ID token** scopes.
```

Comment:
I would suggest making this more general than just B2C specific. i.e. `Why is the idToken or accessToken field in the response empty?` and and move this into either the `Tokens` section or the `Scopes and Resources` section.

Reply:
I had the impression that this is a quirk with the B2C server. Will confirm with AAD sample and revise accordingly 
cc @hectormmg

Reply:
done!